### PR TITLE
test-netconnection: add page

### PR DESCRIPTION
**Pull request by harygary**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/windows/test-netconnection.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Test-NetConnection
+
+> Displays diagnostic information for a connection.
```
Comment (acuteenvy) [inline pages/windows/test-netconnection.md:3]: [Please use the imperative in descriptions](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood).
```suggestion
> Display diagnostic information for a connection.
```

File: pages/windows/test-netconnection.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+> This command can only be used through PowerShell.
+> More information: <https://learn.microsoft.com/powershell/module/nettcpip/test-netconnection>.
+
+- Test a connection and display detailed results:
+
+`Test-NetConnection -InformationLevel Detailed`
+
+- Test a connection to a remote host with port number:
```
Comment (acuteenvy) [inline pages/windows/test-netconnection.md:11]:
```suggestion
- Test a connection to a remote host using the specified port number:
```

File: pages/windows/test-netconnection.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+`Test-NetConnection -ComputerName {{IP or Host Name}} -Port {{Port Number}}`
```
Comment (acuteenvy) [inline pages/windows/test-netconnection.md:13]: > Use [`snake_case`](https://wikipedia.org/wiki/snake_case) for multi-word placeholders.

https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming

```suggestion
`Test-NetConnection -ComputerName {{ip_or_hostname}} -Port {{port_number}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**